meta-iotlab/recipes-devtools/openocd/openocd-ti.bb

```diff
@@ -0,0 +1,76 @@
+DESCRIPTION = "TI's fork of OpenOCD Free and Open On-Chip Debugging, In-System Programming and Boundary-Scan Testing"
+HOMEPAGE = "git://git.ti.com/sdo-emu/openocd.git"
+SECTION = "utils"
+LICENSE = "GPLv2"
+LIC_FILES_CHKSUM = "file://COPYING;md5=b234ee4d69f5fce4486a80fdaf4a4263"
+
+PR = "r1"
+
+DEPENDS = ""
+DEPENDS += "libftdi"
+DEPENDS += "libusb1"
+DEPENDS += "jimtcl"
+DEPENDS += "hidapi"
+DEPENDS += "texinfo-native"
+
+## ##################################################
+## ##################################################
+
+# Use git version 0.10.0: issues when using source code
+SRCREV = "4765fd4c864c47d48754350c196bd5c5ce9c5ed9"
+
+SRC_URI = "git://git.ti.com/sdo-emu/openocd.git;protocol=git"
+
+S = "${WORKDIR}/git/openocd"
+
+## ##################################################
+## ##################################################
+
+PACKAGES = "${PN}-dbg ${PN}"
+
+FILES_${PN}-dbg += "/opt/ti-openocd-0.10.0/.debug"
```

Comment:
for consistency with other paths, maybe put this openocd version ni `/opt/openocd-ti` ?

Reply:
fine by me, will fix